app/scripts/components/facets/facets.controllers.ts

```diff
@@ -98,6 +99,10 @@ module ngApp.components.facets.controllers {
       this.FacetService.addTerm(facet, term);
     }
 
+    addMissing(facet: string): void {
+      this.FacetService.addTerm(facet, 'MISSING');
```

Comment:
`vital_status = blood or vital_status is missing` is already supported on the server side. The adv. search generates the correct filter and the api can consume that data structure. This PR adds the support to `missing` in the `in` filter - therefore the `this.FacetService.addTerm` call @cy mentioned is to add `missing` to the `in` array. Does that make sense?